src/ar/Tag.cpp

```diff
@@ -1,175 +1,29 @@
 #include "Tag.h"
-#include "../Util.h"
-
-#include <opencv2/calib3d.hpp>
-
-#include <cassert>
-#include <iostream>
-#include <cmath>
 
 namespace AR
 {
-double findAngle(cv::Point a, cv::Point b, cv::Point c)
```

Comment:
I guess all this code is deleted because you switched to the OpenCV implementation?

Reply:
Yeah, partly that and partly because we wanted the Tag to be a simpler data structure that just holds a position, rotation, etc. Previously we had the Tag computing its own orientation when you constructed it, which meant that you needed to give it image coordinates of the corners to construct it. That meant we couldn't really use it in the simulator or in testing because we required that you actually detect a tag in an image to make a Tag object. We decided that was more of a job for the detector and that the Tag should just be a simple data structure for returning results from the detector, so all that code was removed; it was replaced in the detector by OpenCV's implementation.